settings.gradle

```diff
@@ -1,7 +1,6 @@
+include ':app', ':dcrlibwallet'
```

Comment:
Revert the changes made to this file. dcrlibwallet is already included in build.gradle as a library and should not be compiled as a project.